libcontainer/cgroups/systemd/unified_hierarchy.go

```diff
@@ -240,10 +240,11 @@ func createCgroupsv2Path(path string) (Err error) {
 					return err
 				}
 			} else {
+				toBeRolledBack := current
```

Comment:
can we please just say
```go
current := current
```
and do it right before `defer` (i.e. after the comment)?